views/includes/check-your-answers/beneficial-owner-gov.html

```diff
@@ -3,8 +3,12 @@
 {% set bogStartDate = dateMacros.formatDate(bog.start_date["day"], bog.start_date["month"], bog.start_date["year"]) %}
 
 {% set bogFormattedPrincipalAddressHtml %}
-  {% set address = bog.principal_address %}
-  {% include "includes/display_address.html" %}
+  {% if (not bog.principal_address.length > 0  and pageParams.noChangeFlag) %}
```

Comment:
why length > 0? bog.principal_address is an object.

Reply:
{%if not bog.principle_address and pageParams.noChangeFlag %}

Reply:
@AndreKramer7654 this was the syntax that existed for the bo other below before that did not work. 

Yes, it is an object meant to user var|length > 0 

https://github.com/mozilla/nunjucks/releases/tag/v2.5.0